iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -146,11 +146,22 @@ def update_special_property(self, name: str) -> bool:
         except KeyError:
             return False
 
-    def attach_special_property(self, name, special_property) -> None:
+    def attach_special_property(
+        self,
+        name: str,
+        special_property: (
+            'iocage.lib.Config.Jail.'
+            'JailConfigProperties.JailConfigProperties'
+        )
+    ) -> None:
         """Attach a special property to the configuration."""
         self.special_properties[name] = special_property
 
-    def _set_legacy(self, value, **kwargs) -> None:
+    def _set_legacy(  # noqa: T484
+        self,
+        value: typing.Union[bool, str],
+        **kwargs
```

Comment:
please open up an issue for cleaning up `noqa: T484`

Reply:
@igalic as you wish #282